deps/openblas.version

```diff
@@ -1,2 +1,4 @@
 OPENBLAS_BRANCH=v0.3.3
 OPENBLAS_SHA1=fd8d1868a126bb9f12bbc43b36ee30d1ba943fbb
+OPENBLAS_VER=0.3.0
```

Comment:
Is this 0.3.0 or 0.3.3

Reply:
Great catch; I'll build `v0.3.3` in [Yggdrasil](https://github.com/JuliaPackaging/Yggdrasil), update this PR, then also submit a PR to bump to `v0.3.4`, since that looks like it [has some important fixes](https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.4).